ui/src/org/pentaho/di/ui/spoon/Spoon.java

```diff
@@ -503,6 +505,12 @@
   // "Redo : not available \tCTRL-Y"
   private static final String REDO_UNAVAILABLE = BaseMessages.getString( PKG, "Spoon.Menu.Redo.NotAvailable" );
 
+  private static boolean UNSUPPORTED_BROWSER_ENVIRONMENT;
```

Comment:
named as a constant but it's not a constant... can be it static final? otherwise better rename it.